app.js

```diff
@@ -58,7 +58,7 @@
     const distance = Math.round(closestSensor.distance * 10) / 10
     const time = (new Date()).toLocaleTimeString()
     const paLink = getPurpleAirLink()
-    const aqiMsg = `AQI is ${aqi}`
+    const aqiMsg = `${getAQIEmoji(aqi)} AQI is ${aqi}`
```

Comment:
```suggestion
    const aqiMsg = `AQI is ${aqi} ${getAQIEmoji(aqi)}`
```

I don't know why but emoji as endings read better to me than having them lead. What do you think?